packages/http-client-java/generator/http-client-generator/src/main/java/com/microsoft/typespec/http/client/generator/model/EmitterOptions.java

```diff
@@ -30,7 +30,7 @@ public class EmitterOptions implements JsonSerializable<EmitterOptions> {
     private String customizationClass;
     private Boolean includeApiViewProperties = true;
 
-    private Boolean useObjectForUnknown = false;
+    private Boolean useObjectForUnknown = true;
```

Comment:
There should be a nominal `use-object-for-unknown` default to `true` in emitter.ts (it won't take effect). Though currently it was `true` there (a mistake).